x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/flyout_components/item_comments/index.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiAccordion, EuiSpacer, EuiTitle } from '@elastic/eui';
+import styled, { css } from 'styled-components';
+import type { Comment } from '@kbn/securitysolution-io-ts-list-types';
+
+import * as i18n from './translations';
+import { ExceptionItemComments } from '../../item_comments';
+
+const SectionHeader = styled(EuiTitle)`
+  ${() => css`
+    font-weight: ${({ theme }) => theme.eui.euiFontWeightSemiBold};
+  `}
+`;
+
+interface ExceptionsFlyoutCommentsProps {
+  existingComments?: Comment[];
+  newComment: string;
+  onCommentChange: (comment: string) => void;
+}
+
+const ExceptionsFlyoutCommentsComponent: React.FC<ExceptionsFlyoutCommentsProps> = ({
+  existingComments,
+  newComment,
+  onCommentChange,
+}): JSX.Element => {
+  return (
+    <EuiAccordion
```

Comment:
Can't we reuse the Comment component defined here `packages/kbn-securitysolution-exception-list-components/src/exception_item_card/comments`? 🤔

Reply:
I think we can! I'll refactor.